controllers/storagecluster/cephobjectstores.go

```diff
@@ -227,6 +227,8 @@ func (r *StorageClusterReconciler) newCephObjectStoreInstances(initData *ocsv1.S
 			obj.Spec.Security = &cephv1.ObjectStoreSecuritySpec{
 				SecuritySpec: cephv1.SecuritySpec{
 					KeyManagementService: cephv1.KeyManagementServiceSpec{
+						EnableKeyRotation: initData.Spec.Security.KeyManagementService.EnableKeyRotation,
```

Comment:
no need to make changes to cephobjectstores... that is out of scope of this epic...